plasmapy/formulary/relativity.py

```diff
@@ -88,3 +87,71 @@ def Lorentz_factor(V: u.m / u.s):
             gamma = ((1 - (V / c) ** 2) ** -0.5).value
 
     return gamma
+
+
+@validate_quantities(m={"can_be_negative": False}, 
+                     validations_on_return={"can_be_negative": False})
+def relativistic_energy(m: u.kg, v: u.m / u.s) -> u.Joule:
+    """
+    Calculate the relativistic energy (in Joules) of an object of mass 
+    `m` and velocity `v`.
+    
+    .. math::
+
+        E = \\gamma m c^{2}
+    
+    where :math:`gamma` is the `Lorentz_factor`.
+
+    Parameters
+    ----------
+    m : `~astropy.units.Quantity`
+        The mass in units convertible to kilograms.
+
+    v : `~astropy.units.Quantity`
+        The velocity in units convertible to meters per second.
+
+    Returns
+    -------
+    `~astropy.Quantity`
+        The relativistic energy (in Joules) of an object of mass `m` moving at velocity `v`.
+
+    Raises
+    ------
+    TypeError
+        If input arguments are not instances `~astropy.units.Quantity` or
+        convertible to a `~astropy.units.Quantity`.
+
+    ~astropy.units.UnitConversionError
+        If the `v` is not in appropriate units.
+
+    ValueError
+        If the magnitude of 'm' is negative.
+
+    `RelativityError`
+        If the velocity `v` is greater than the speed of light.
+
+    Warns
+    -----
+    `~astropy.units.UnitsWarning`
```

Comment:
```suggestion
    : `~astropy.units.UnitsWarning`
```

It appears the **Warns** section follows the formatting of the **Parameters** section, opposed to the **Raises** section.